gcloud-java-core/src/test/java/com/google/gcloud/RetryHelperTest.java

```diff
@@ -203,6 +205,31 @@ public void testTriesNoMoreLongerThanTotalRetryPeriod() {
   }
 
   @Test
+  public void testNoRetriesOnSocketTimeout() {
+    final FakeTicker ticker = new FakeTicker();
+    Stopwatch stopwatch = Stopwatch.createUnstarted(ticker);
+    RetryParams params = RetryParams.builder().initialRetryDelayMillis(0)
+        .totalRetryPeriodMillis(999)
+        .retryMinAttempts(5)
+        .retryMaxAttempts(10)
+        .build();
+    ExceptionHandler handler = ExceptionHandler.builder().retryOn(RuntimeException.class).build();
+    final AtomicInteger timesCalled = new AtomicInteger(0);
+    try {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'RetryHelper.runWithRetries' due to existing static import 'com.google.gcloud.RetryHelper.runWithRetries'](https://www.codacy.com/app/mziccard/gcloud-java/file/2381071151/issues/source?bid=3123103&fileBranchId=3225812#l219)